<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Udp2</class>
 <widget class="QWidget" name="Udp2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>杨义杰</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labMyPort">
        <property name="text">
         <string>MyPort</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="MyPort"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labToPort">
        <property name="text">
         <string>ToPort</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ToPort"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labToIP">
        <property name="text">
         <string>ToIP</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ToIP"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>聊天记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="record"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>输入内容</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="input"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonSend">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
